ncloud/resource_ncloud_lb_target_group.go

```diff
@@ -0,0 +1,330 @@
+				Required: true,
+			},
+			"use_sticky_session": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"use_proxy_protocol": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
+			},
+			"algorithm_type": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Computed:         true,
+				ValidateDiagFunc: ToDiagFunc(validation.StringInSlice([]string{"RR", "SIPHS", "LC", "MH"}, false)),
+			},
+			"load_balancer_instance_no": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceNcloudTargetGroupCreate(d *schema.ResourceData, meta interface{}) error {
+	config := meta.(*ProviderConfig)
+
+	reqParams := &vloadbalancer.CreateTargetGroupRequest{
+		RegionCode: &config.RegionCode,
+		// Optional
+		TargetGroupPort:        Int32PtrOrNil(d.GetOk("port")),
+		TargetGroupDescription: StringPtrOrNil(d.GetOk("description")),
+		TargetGroupName:        StringPtrOrNil(d.GetOk("name")),
+		TargetNoList:           StringListPtrOrNil(d.GetOk("target_no_list")),
+		// Required
+		TargetTypeCode:              ncloud.String(d.Get("target_type").(string)),
+		VpcNo:                       ncloud.String(d.Get("vpc_no").(string)),
+		TargetGroupProtocolTypeCode: ncloud.String(d.Get("protocol").(string)),
+	}
+
+	if healthChecks, ok := d.GetOk("health_check"); ok {
+		healthCheck := healthChecks.([]interface{})[0].(map[string]interface{})
+
+		reqParams.HealthCheckCycle = ncloud.Int32(int32(healthCheck["cycle"].(int)))
+		reqParams.HealthCheckDownThreshold = ncloud.Int32(int32(healthCheck["down_threshold"].(int)))
+		reqParams.HealthCheckUpThreshold = ncloud.Int32(int32(healthCheck["up_threshold"].(int)))
+		reqParams.HealthCheckPort = ncloud.Int32(int32(healthCheck["port"].(int)))
+
+		// Required
+		reqParams.HealthCheckProtocolTypeCode = ncloud.String(healthCheck["protocol"].(string))
+		if err := validateHealthCheckProtocolByTargetGroupProtocol(*reqParams.TargetGroupProtocolTypeCode, *reqParams.HealthCheckProtocolTypeCode); err != nil {
+			return err
+		}
+
+		if *reqParams.HealthCheckProtocolTypeCode == "HTTP" || *reqParams.HealthCheckProtocolTypeCode == "HTTPS" {
+			reqParams.HealthCheckUrlPath = ncloud.String(healthCheck["url_path"].(string))
+			if healthCheck["http_method"] == "" {
+				return fmt.Errorf("http_method is required if the health check protocol type is HTTP or HTTPS.")
+			}
+			reqParams.HealthCheckHttpMethodTypeCode = ncloud.String(healthCheck["http_method"].(string))
+		}
+	}
+
+	logCommonRequest("resourceNcloudTargetGroupCreate", reqParams)
+	resp, err := config.Client.vloadbalancer.V2Api.CreateTargetGroup(reqParams)
+	logResponse("resourceNcloudTargetGroupCreate", resp)
+	if err != nil {
+		logErrorResponse("resourceNcloudTargetGroupCreate", err, reqParams)
+		return err
+	}
+
+	d.SetId(ncloud.StringValue(resp.TargetGroupList[0].TargetGroupNo))
+	return resourceNcloudTargetGroupRead(d, meta)
```

Comment:
use_sticky_session, use_proxy_protocol  관련 로직을 위해 아래와 같이 변경 되어야 할 것 같습니다.

```suggestion
	return resourceNcloudTargetGroupUpdate(d, meta)
```

Reply:
네 반영하였습니다!